ui/src/turing/components/form/standard_ensembler/LinkedExperimentsContextMenu.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+
+import {
+  EuiContextMenu,
+  EuiPopover,
+  EuiButtonEmpty,
+  EuiLink,
+  EuiIcon
+} from "@elastic/eui";
+import { useToggle } from "@gojek/mlp-ui";
+
+export const LinkedExperimentsContextMenu = ({
+  projectId,
+  linkedExperiments,
+  experimentStatus
+}) => {
+  const [isPopoverOpen, togglePopover] = useToggle();
+
+  let numExperiments = linkedExperiments ? Object.keys(linkedExperiments[experimentStatus]).length : 0;
+
+  const button = (
+    <EuiButtonEmpty
+      size={"s"}
+      iconType={"arrowRight"}
+      iconSide={"right"}
+      color={"primary"}
+      onClick={togglePopover}
+      isDisabled={numExperiments === 0}
+    >
+      {numExperiments}
+    </EuiButtonEmpty>
+  );
+
+  return linkedExperiments ? (
```

Comment:
In relation to a previous thread - https://github.com/caraml-dev/xp/pull/37#discussion_r978387554, since the rendering is dependent on `linkedExperiments` which is passed down from the parent component, why not do the following instead? Then, we won't need the ternary for this return statement.
```
render: (id) => routeToExperimentMappings[id] && (
        <LinkedExperimentsContextMenu
          projectId={projectId}
          linkedExperiments={routeToExperimentMappings[id]}
          experimentStatus={"scheduled"}
        />
      )
```

Reply:
Thanks for the suggestion! Just made the change!